### PR TITLE
Attempted to fully document all possible settings in config files

### PR DESCRIPTION
**Pull request by ruschein**
used by convert_json_to_marc.

### CODE REVIEW

File: cpp/data/convert_json_to_marc.README [hunk 1]
```diff
@@ -22,13 +22,27 @@ item_type_map[O]: If specified, item_type_tag must be also specified.  This cons
...
 
 2. The MARC-21 Field Section
-indicatior1[O]  = the first indicator for the generated field if it is a data field.  The default is a blank.
-indicatior2[O]  = the second indicator for the generated field if it is a data field.  The default is a blank.
-tag[R]          = the MARC tag of the MARC field that the extracted JSON will be written to
-overflow_tag[O] = if specified the first value will be stored in a MARC field with tag "tag" and subsequent
-                  values are stored in MARC fields with tag "overflow_tag", e.g. 100 and 700 for authors.
-normalise_issn  = if true, values of the form XXXX-YYYY will be stored as XXXXYYYY, values that started out
-                  without the hyphen will be unaffected.
+indicatior1[O]             = the first indicator for the generated field if it is a data field.  The default is a blank.
```
Comment (mtrojan-ub) [inline cpp/data/convert_json_to_marc.README]: spelling mistake "indicat**i**or"

File: cpp/data/convert_json_to_marc.README [hunk 2]
```diff
@@ -22,13 +22,27 @@ item_type_map[O]: If specified, item_type_tag must be also specified.  This cons
...
+                             section
+required                   = the default, if this entry is not present, is "false".  If present and "true", a
+                             warning is issued if the referenced JSON data is missing
+subfield_x_prefix          = if present, the value of this entry will be prepended to values inserted into
+                             subfield "x"
+add_fixed_subfield_x       = if subfields are generated at all for a field then the value of this entry will be
+                             used to generate an additional subfield "x"
+map_to_marc_language_code = if present, an attempt will be made to map what will be inserted into a subfield
+                            to a language code from the MARC standard.  If the mapping fails nothing will be
+                            inserted
```
Comment (mtrojan-ub) [inline cpp/data/convert_json_to_marc.README]: What about:
- json_tag
- repeat_field

File: cpp/data/convert_json_to_marc.README [hunk 3]
```diff
@@ -22,13 +22,27 @@ item_type_map[O]: If specified, item_type_tag must be also specified.  This cons
...
-indicatior1[O]  = the first indicator for the generated field if it is a data field.  The default is a blank.
-indicatior2[O]  = the second indicator for the generated field if it is a data field.  The default is a blank.
-tag[R]          = the MARC tag of the MARC field that the extracted JSON will be written to
-overflow_tag[O] = if specified the first value will be stored in a MARC field with tag "tag" and subsequent
-                  values are stored in MARC fields with tag "overflow_tag", e.g. 100 and 700 for authors.
-normalise_issn  = if true, values of the form XXXX-YYYY will be stored as XXXXYYYY, values that started out
-                  without the hyphen will be unaffected.
+indicatior1[O]             = the first indicator for the generated field if it is a data field.  The default is a blank.
+indicatior2[O]             = the second indicator for the generated field if it is a data field.  The default is a blank.
+tag[R]                     = the MARC tag of the MARC field that the extracted JSON will be written to
```
Comment (mtrojan-ub) [inline cpp/data/convert_json_to_marc.README]: what does [R], [0] ... mean?

Comment (ruschein) [reply cpp/data/convert_json_to_marc.README]: See the bottom of the file

**Pull request merged**